### PR TITLE
Pol 174 update google policies for credential service 2

### PR DESCRIPTION
**Pull request by gopinathmahakud1**
### Description

- Replace auth declaration with the credential declaration
- Remove long_description field and use new info hash for Version, providers and service.
- Update readme to replace credential instructions with instructions to use credential service.
- Update CWF definitions as needed.
- Add info block
- Remove the Google Project param use the Projects API to get projects ID

### Issues Resolved

- Replaced auth declaration with the credential declaration
- Removed long_description field and use new info hash for Version, providers and service.
- Updated readme to replace credential instructions with instructions to use credential service.
- Updated CWF definitions as needed.
- Added info block
- Removed the Google Project param used the Projects API to get projects ID

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5e42a1fecbdfbd000119ae15
https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5e439720cbdfbd000119afbc
https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5e43a01fcbdfbd000119afc9

### CODE REVIEW

File: cost/google/cud_expiration/README.md [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
 
-### What it does
 This policy identifies all active CUDs that exist in a given GCP project that will be expiring in a set number of days.
 
-### Functional Details
+## Functional Details
 
 - Uses the GCP API to get a list of all CUDs and sends a notification.
-- Create a service account (if none exist) with `owner` role under Google-cloud platform (IAM & admin -> service accounts). Generate key, a JSON file will get downloaded in which you can find 'client email' and 'private key' which has to be added as credentials in RightScale cloud management Design -> Credentials with name 'GC_SA_CLIENT_EMAIL' and 'GC_SA_PRIVATE_KEY' respectively.  
+- Create a service account (if none exist) with `owner` role under Google-cloud platform (IAM & admin -> service accounts). Generate key, a JSON file will get downloaded in which you can find 'client email' and 'private key' which has to be added as credentials in RightScale cloud management Design -> Credentials with name 'GC_SA_CLIENT_EMAIL' and 'GC_SA_PRIVATE_KEY' respectively.
```
Comment (cdwilhelm) [inline cost/google/cud_expiration/README.md]: This part is no longer needed.

File: cost/google/cud_report/README.md [hunk 2]
```diff
@@ -1,39 +1,38 @@
...
 
-### What it does
 This policy identifies all CUDs that exist in a given GCP project and sends a notification. It can optionally send a notification on all CUDs or only those that are active or expired.
 
-### Functional Details
+## Functional Details
 
 - Uses the GCP API to get a list of all CUDs and sends a notification.
-- Create a service account (if not exists) with owner role under Google-cloud platform (IAM & admin -> service accounts). Generate key, a JSON file will get downloaded in which you can find 'client email' and 'private key' which has to be added as credentials in RightScale cloud management Design -> Credentials with name 'GC_SA_CLIENT_EMAIL' and 'GC_SA_PRIVATE_KEY' respectively.  
+- Create a service account (if not exists) with owner role under Google-cloud platform (IAM & admin -> service accounts). Generate key, a JSON file will get downloaded in which you can find 'client email' and 'private key' which has to be added as credentials in RightScale cloud management Design -> Credentials with name 'GC_SA_CLIENT_EMAIL' and 'GC_SA_PRIVATE_KEY' respectively.
```
Comment (cdwilhelm) [inline cost/google/cud_report/README.md]: This part is no longer needed

File: cost/google/unutilized_ip_addresses/README.md [hunk 3]
```diff
@@ -1,31 +1,48 @@
...
 
-### Cloud Management Required Permissions/Google Required Permissions
-- Cloud Management - The `credential_viewer`, `observer` roles
-- Cloud Management - The `policy_designer`, `policy_manager` & `policy_publisher` roles
-- Google - The `compute.addresses.list`, `compute.addresses.get`, and `compute.addresses.delete` IAM Permissions
+## Functional Details
 
-### Functional Details
+- This policy uses Google Cloud to get a list of IP addresses, internal and external, that are not in use.
+- Create a service account (if not exists) with the necessary permissions under Google-cloud platform (IAM & admin -> service accounts). Generate key, a JSON file will get downloaded in which you can find 'client email' and 'private key' which has to be added as credentials in RightScale cloud management Design -> Credentials with name 'GCE_PLUGIN_ACCOUNT' and 'GCE_PLUGIN_PRIVATE_KEY' respectively.
```
Comment (cdwilhelm) [inline cost/google/unutilized_ip_addresses/README.md]: This is no longer needed

File: cost/google/unutilized_ip_addresses/README.md [hunk 4]
```diff
@@ -1,31 +1,48 @@
-## Google Unutilized IP Addresses Policy
+# Google Unutilized IP Addresses Policy
```
Comment (cdwilhelm) [inline cost/google/unutilized_ip_addresses/README.md]: please remove the work Policy

File: cost/google/unutilized_ip_addresses/README.md [hunk 5]
```diff
@@ -1,31 +1,48 @@
...
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
+
+Provider tag value to match this policy: `gce`
+
+Required permissions in the provider:
+
+- The `compute.addresses.delete` permission
+- The `compute.addresses.list` permission
+- The `compute.addresses.get` permission
+- The 'compute.regions.list' permission
```
Comment (cdwilhelm) [inline cost/google/unutilized_ip_addresses/README.md]: Use back tick instead of quote

File: cost/google/cud_expiration/README.md (hunk 1 above)
Comment (gopinathmahakud1) [reply cost/google/cud_expiration/README.md]: I have addressed the comment. Please review the changes.

File: cost/google/cud_report/README.md (hunk 2 above)
Comment (gopinathmahakud1) [reply cost/google/cud_report/README.md]: I have addressed the comment. Please review the changes.

File: cost/google/unutilized_ip_addresses/README.md (hunk 4 above)
Comment (gopinathmahakud1) [reply cost/google/unutilized_ip_addresses/README.md]: I have addressed the comment. Please review the changes.

File: cost/google/unutilized_ip_addresses/README.md (hunk 5 above)
Comment (gopinathmahakud1) [reply cost/google/unutilized_ip_addresses/README.md]: I have addressed the comment. Please review the changes.

File: cost/google/unutilized_ip_addresses/README.md (hunk 3 above)
Comment (gopinathmahakud1) [reply cost/google/unutilized_ip_addresses/README.md]: I have addressed the comment. Please review the changes.

**Pull request merged**